simple_salesforce/api.py

```diff
@@ -565,12 +571,8 @@ def _call_salesforce(self, method, url, **kwargs):
 
         Returns a `requests.result` object.
         """
-        headers = {
-            'Content-Type': 'application/json',
-            'Authorization': 'Bearer ' + self.session_id,
-            'X-PrettyPrint': '1'
-        }
-        result = self.request.request(method, url, headers=headers, **kwargs)
+
+        result = self.request.request(method, url, headers=self.headers, **kwargs)
```

Comment:
Do lines 414-418 take care of that on the __init__ of the sftype? 

        self.headers = headers if headers else {
              'Content-Type': 'application/json',
              'Authorization': 'Bearer ' + self.session_id,
              'X-PrettyPrint': '1'
          }

I can add in a header re-definition/check back into __call_salesforce, just wanted to double check I wasn't missing something.

# contact w/ default headers specified in connection
my_contact = sf.Contact

# leads w/ custom header
sfleads = SFType('Lead', sf.session_id, sf.sf_instance, sf.proxies, mycustomheaders)